src/olympia/devhub/templates/devhub/validation.html

```diff
@@ -24,8 +24,10 @@ <h2>{{ title }}</h2>
 
 <div class="addon-validator-suite"
      id="addon-validator-suite"
-     {% if file %}
+     {% if file_url %}
```

Comment:
I was going to suggest changing this to something more descriptive (and different from `file`) like `code_review_url` but it's old code and hopefully going to be changed soon anyway so 🤷‍♂️